packages/gatsby-cli/src/init-starter.js

```diff
@@ -67,8 +71,30 @@ const install = async rootPath => {
   process.chdir(rootPath)
 
   try {
-    let cmd = shouldUseYarn() ? spawn(`yarnpkg`) : spawn(`npm install`)
-    await cmd
+    const npmCmd = `npm install`
+    let response = npmCmd
+    if (shouldUseYarn()) {
+      const promptsAnswer = await prompts([
```

Comment:
Will this wait infinitely long for user-input?

Reply:
Yes until user select one choice